packages/effect-plugin/package.json

```diff
@@ -13,6 +13,25 @@
     "test": "jest",
     "lint": "set -ex; tsc --build; tslint --project ."
   },
+  "homepage": "https://github.com/prodo-ai/prodo/tree/master/packages/effect-plugin#readme",
+  "bugs": {
+    "url": "https://github.com/prodo-ai/prodo/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/prodo-ai/prodo.git",
+    "directory": "packages/effect-plugin"
+  },
+  "keywords": [
+    "prodo",
+    "state-management",
+    "react",
+    "redux",
+    "mobx",
+    "effects",
+    "flux",
+    "prodo-eslint"
```

Comment:
Did you mean to have prodo-eslint in this one? (or was it just meant for the esling-plugin?)

Reply:
Oops, you are right.